api-tests/src/test/java/org/openmrs/module/reporting/evaluation/EvaluationProfilerTest.java

```diff
@@ -33,6 +38,8 @@
 /**
  * Tests for {@link EvaluationProfiler}
  */
+@TestExecutionListeners(OpenmrsVersionTestListener.class)
+@RequiresVersion("1.* - 2.3.*")
```

Comment:
This test no longer works on 2.4 because of the migration to Log4J2 which has a very similar interface for logging, but a very different one to manage logging. This is also why creating an api-tests-2.4 module seemed necessary.

Reply:
Sounds good.  We could also potentially just remove the EvaluationProfiler altogether, or add compile-time dependencies on a specific version of log4j, etc.  I'm fine with what you did, but all of the different maven submodules does start to add up the complexity.